### ISSUE TITLE
Feature request: when region is active, show word/line/char count in region

### ISSUE THREAD

**Issue by duetosymmetry** [bnbeckwith/wc-mode#10]
The strings %rw, %rl, and %rc could insert (i) an emptry string if no region is active, or (ii) the word/line/char count in the active region

**Comment by sten0** [comment]
`wc` (aliased `wc-count`) can operate on region.  Do you mean using the modeline for a region-local value rather than a buffer-local one?

**Comment by duetosymmetry** [comment]
Yes, it would be great to use the modeline to see the currently active region's words/lines/chars.
